packages/cfd/src/Assets/svgs/trading-instruments/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import DerivedFX from './ic-appstore-derived-fx.svg';
+import Synthetics from './ic-appstore-synthetics.svg';
+import BasketIndices from './ic-appstore-basket-indices.svg';
+import Stocks from './ic-appstore-stocks.svg';
+import StockIndices from './ic-appstore-stock-indices.svg';
+import Commodities from './ic-appstore-commodities.svg';
+import Forex from './ic-appstore-forex.svg';
+import Cryptocurrencies from './ic-appstore-cryptocurrencies.svg';
+import ETF from './ic-appstore-etf.svg';
+
+export interface IconProps<T> {
```

Comment:
Try to restrict this generic type. T accepts all possible typescript types this way.